test/mocks/upstream/transport_socket_match.cc

```diff
@@ -0,0 +1,26 @@
+#include "test/mocks/upstream/transport_socket_match.h"
+
+#include "common/network/raw_buffer_socket.h"
+
+using testing::_;
+using testing::Invoke;
+using testing::Return;
+
+namespace Envoy {
+namespace Upstream {
+namespace Outlier {
```

Comment:
erh, why this in Outlier??

Reply:
oh, i was confused as well before. copied from other part, now I realized that only enclose portion of the header files.

Removed the outlier.